<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Computer Vision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="pluginGroup">
        <property name="title">
         <string>Plugin UI</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="pluginLayout"/>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="PdwVisionLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="openLineEdit"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="openPushButton">
                    <property name="text">
                     <string>Open</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/btn/Image/open.bmp</normaloff>:/btn/Image/open.bmp</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="connectSimulatorPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>200</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Connect Simulator</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/btn/Image/select.bmp</normaloff>:/btn/Image/select.bmp</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="runPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>200</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Run</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/btn/Image/run.bmp</normaloff>:/btn/Image/run.bmp</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="stopPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>200</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Stop</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/btn/Image/stop.bmp</normaloff>:/btn/Image/stop.bmp</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Pixel Unit : </string>
             </property>
             <property name="buddy">
              <cstring>lineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="viewOriginalCheck">
             <property name="text">
              <string>View Original Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QEnhancedGraphicsView" name="graphicsView"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionTheme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QEnhancedGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>qenhancedgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
